HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,48 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is the platform specific path separator.
+
+method::isVowel
+
+returns:: true if the char is a vowel.
+
+method::isControl
+
+returns:: true if the char is a control character.
+
+method::asAscii
+
+returns:: the object itself.
+
+method::asUnicode
+
+returns:: a single Unicode value. Ascii is a subset of Unicode.
+
+method::hash
+
+returns:: the hash value of the receiver.
+
+method::++
+
+A concatenation of the two chars. The result is link::Classes/String::
+
+method::<
+
+returns:: true if the Unicode value of the receiver is less that the Unicode value of the argument. While this is equivalent to alphabetical order when comparing for example two lowercase characters, uppercase characters have lower Unicode values that lowercase characters, and digits have lower values than either.
```

Comment:
Unfortunately doesn't make a lot of sense because of Char not really having a proper encoding. I would document this as simply `this.ascii < aChar.ascii`. Uppercase characters comparing less than lowercase doesn't apply in most encodings once you add in diacritics (ĕ is codepoint U+0115, Ã is U+00C3 for example), you may want to specify that you're talking about comparing ASCII values specifically.

Reply:
ok. thanks